### PR TITLE
Small tweaks for CRAN

### PR DESCRIPTION
**Pull request by mattsecrest**
- [x] missed a DOI previously
- [x] edge case where cmdstanr is installed but cmdstan is not

### CODE REVIEW

**Comment by mattsecrest** [comment]
@danielinteractive @gravesti some very light updates to take away one more NOTE and to fix an edge case where cmdstanr is isntalled but cmdstan is not (r-hub was encountering this).

**Comment by mattsecrest** [comment]
image

File: R/borrowing_hierarchical_commensurate.R [hunk 1]
```diff
@@ -42,10 +42,9 @@
 #' In Bayesian dynamic borrowing using the hierarchical commensurate prior approach,
 #' external control information is borrowed to the extent that the
 #' outcomes (i.e., log hazard rates or log odds) are similar between
-#' external and internal control populations. See
-#' \href{https://doi.org/10.1002/pst.1589}{Viele et. al. 2014} and
-#' \href{https://doi.org/10.1111/j.1541-0420.2011.01564.x}{Hobbs et. al. 2011}
-#' for details.
+#' external and internal control populations. See Viele 2014
+#' \doi{10.1002/pst.1589} and Hobbs 2011
```
Comment (danielinteractive) [inline R/borrowing_hierarchical_commensurate.R:46]: minor comment for the future: these references would be easier written when using`Rdpack` (https://geobosh.github.io/Rdpack/)

**Review by danielinteractive — APPROVED** [review]

File: R/borrowing_hierarchical_commensurate.R (hunk 1 above)
Comment (mattsecrest) [reply R/borrowing_hierarchical_commensurate.R:46]: Good call, thanks @danielinteractive

**Pull request merged**